cuda/image_basic_op/CMakeLists.txt

```diff
@@ -18,11 +18,12 @@ set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -allow-unsupported-compiler --extended
 set(CMAKE_CUDA_RUNTIME_LIBRARY "Static")
 
 target_compile_definitions(tipl_image_basic_op PUBLIC TIPL_USE_CUDA)
-
-if(EXISTS "${TIPL_DIR}")
-  include_directories(tipl_image_basic_op ${TIPL_DIR})
+find_package(TIPL)
```

Comment:
This way if TIPL_DIR corresponds to ..../lib/cmake/ with TIPLConfig.cmake in it, find_package will succeed. if not we can use TIPL_DIR to point to the toplevel TIPL directory